riff-raff/public/docs/reference/riff-raff.yaml.md

```diff
@@ -144,21 +149,35 @@ Examples
 In order to understand how Riff-Raff interprets these files feel free to copy the YAML and paste it into Validate
 Template page in Riff-Raff.
 
-Here is a minimal auto scaling deploy example taken from [Prism](https://github.com/guardian/prism/blob/master/riff-raff.yaml).
+Here is an example of the configuration for an EC2-based application, where Riff-Raff is responsible for deploying
+the infrastructure (via CloudFormation) as well as application code changes:
 
 ```yaml
-regions:
-- eu-west-1
 stacks:
-- deploy
+  - deploy
+regions:
+  - eu-west-1
 deployments:
-  prism:
+  amigo:
     type: autoscaling
     parameters:
-      bucket: deploy-tools-dist
+    dependencies:
+      - cloudformation
+  cloudformation:
+    type: cloud-formation
+    app: amigo
+    parameters:
+      amiTags:
+        Recipe: arm64-jammy-java21-deploy-infrastructure
+        AmigoStage: PROD
+      amiEncrypted: true
+      templateStagePaths:
+        CODE: AMIgo-CODE.template.json
+        PROD: AMIgo-PROD.template.json
```

Comment:
I might be mistaken, but usage of `templateStagePaths` is limited to CDK defined templates as we produce a distinct template per stage. CFN YAML templates use CFN conditions on a `Stage` parameter to achieve stage awareness. If this section is providing an example of a hand-written `riff-raff.yaml`, should we tweak the example to be CFN YAML focused?